docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -68,30 +68,40 @@ You are now signed in to Semgrep App.
-    <dt>Read permissions to <a href="https://docs.github.com/en/rest/reference/checks">GitHub Checks</a></dt>
-    <dd>Facilitates debugging of Semgrep App when configured out of GitHub Actions.</dd>
-    <dt>Read and write to GitHub Security Events</dt>
-    <dd>Enables integration with the GitHub Advanced Security to show Semgrep results.</dd>
-    <dt><code>.semgrepignore</code> single-file access</dt>
-    <dd>Allows debugging of requests and automatic syncing of <code>.semgrepignore</code> between the Semgrep App UI and what is checked into the repository.</dd>
-    <dt>Read/write GitHub secrets</dt>
-    <dd>Enables automatically adding of the Semgrep App Token to your repository secrets when onboarding projects. This simplifies bulk onboarding of repositories. <strong>Semgrep App cannot read the values of your existing or future secrets due to the security design of GitHub Secrets.</strong> This permission only permits Semgrep App to know the secret name and programmatically adds the Semgrep token to your repository secrets.</dd>
+    <dt>Reading metadata of the repositories you select</dt>
+    <dd>Enables Semgrep App to list repository names on the project setup page.</dd>
+    <dt>Reading the list of organization members</dt>
+    <dd>Enables Semgrep App to determine who can manage your Semgrep organization based on your GitHub organization's members list.</dd>
+    <dt>Reading and writing pull requests</dt>
+    <dd>Enables Semgrep App to comment about findings on pull requests.</dd>
+    <dt>Reading and writing actions</dt>
+    <dd>Enables Semgrep App to cancel stuck jobs, rerun jobs, pull logs from jobs, and perform on-demand scanning.</dd>
+    <dt>Reading <a href="https://docs.github.com/en/rest/reference/checks">GitHub Checks</a></dt>
+    <dd>Facilitates debugging of Semgrep App when configured out of <a href="https://docs.github.com/en/actions">GitHub Actions</a>.</dd>
+    <dt>Reading and writing security events</dt>
+    <dd>Enables integration with GitHub Advanced Security (for example, to show Semgrep results).</dd>
+    <dt>Reading and writing secrets</dt>
+    <dd>Enables automatically adding of the Semgrep App Token to your repository secrets when onboarding projects. Note: We cannot read the values of your existing or future secrets (only the names).</dd>
+    <dt>Writing (and reading) 2 files</dt>
+    <dd>Enables Semgrep App to configure itself to run in CI by writing to <code>.github/workflows/semgrep.yml</code> and <code>.semgrepignore</code>.</dd>
+    <dt>Writing (and reading) workflows</dt>
+    <dd>Enables Semgrep App to configure itself to run in CI by writing to <code>.github/workflows/semgrep.yml</code>. GitHub allows writing to files within <code>.github/workflows/</code> only if this permission is granted along with "Writing a single file".</dd>
+    <dt>Reading PRs, a single file, and workflows</dt>

```

Comment:
An interesting anacoluthon maybe? - all items should be plural (Reading PRs, single files, and workflows). But I guess this is how GitHub named this permission and we cannot change it. Feel free to resolve and ignore this comment if this cannot be changed.

Reply:
I'll resolve this discussion and propose a complete change to these two lines